merchstore/models.py

```diff
@@ -1,40 +1,90 @@
 from django.db import models
 from django.urls import reverse
+#from user_management.models import Profile
 
 
 class ProductType(models.Model):
 
     name = models.CharField(max_length=255, unique=True)
     description = models.TextField()
 
+
     class Meta:
         ordering = ["name"]
-        verbose_name = "ProductType"
-        verbose_name_plural = "ProductTypes"
+        verbose_name = "Product Type"
+        verbose_name_plural = "Product Types"
+
 
     def __str__(self):
         return self.name
 
+    #May get deleted...unless there's still time
+    '''
     def get_absolute_url(self):
         return reverse("merchstore_variety", kwargs={"product_type": self.name})
+    '''
 
 
 class Product(models.Model):
+    STATUS_CHOICES = {
```

Comment:
For choices in a `CharField` or `TextField` follow this example:

```python
    class CommissionStatusOptions(models.TextChoices):
        OPEN = "Open"
        FULL = "Full"
        COMPLETED = "Completed"
        DISCONTINUED = "Discontinued"
```

This will be a class inside `Product`. Reference: https://docs.djangoproject.com/en/5.1/ref/models/fields/#enumeration-types